### PR TITLE
Development to main, week 36

### PR DESCRIPTION
**Pull request by remko48**

### CODE REVIEW

**Review by lencodes — CHANGES_REQUESTED** [review]
Can you request a re-review once the requested changes in https://github.com/ConductionNL/conduction-components/pull/103 are implemented?

**Comment by remko48** [comment]
> Can you request a re-review once the requested changes in [ConductionNL/conduction-components#103](https://github.com/ConductionNL/conduction-components/pull/103) are implemented?

Requested changes have been implemented.
Merged to main after getting verbal Approval

**Pull request merged**
